spec/system/candidate_interface/offers_and_withdrawals/candidate_receives_rejection_email_spec.rb

```diff
@@ -53,7 +52,7 @@ def when_i_have_a_single_offer
     @offer = create(:application_choice,
                     status: :offer,
                     application_form: @application_form,
-                    decline_by_default_at: 10.business_days.from_now,
+                    decline_by_default_at: 10.business_days.from_now.midday,
```

Comment:
irl I think `TimeLimitCalculator` sets this to end_of_day?

Reply:
The `:with_offer` trait ought to set this, but the DBD looks a bit dodgy

https://github.com/DFE-Digital/apply-for-teacher-training/blob/9a3cb721f5f9d6204259a9d4569ecf66d0ab0ee2/spec/factories/application_choice.rb#L155-L161

Reply:
Yea good spot. This should be using business days. Will sort now 👍